arch/risc-v/src/esp32c3/esp32c3_rtc.c

```diff
@@ -210,18 +214,26 @@
  */
 
 #define RTC_CONFIG_DEFAULT() {\
-    .ck8m_wait = RTC_CNTL_CK8M_WAIT_DEFAULT, \
-    .xtal_wait = RTC_CNTL_XTL_BUF_WAIT_DEFAULT, \
-    .pll_wait  = RTC_CNTL_PLL_BUF_WAIT_DEFAULT, \
-    .clkctl_init = 1, \
-    .pwrctl_init = 1, \
-    .rtc_dboost_fpd = 1, \
-    .xtal_fpu = 0, \
-    .bbpll_fpu = 0, \
-    .cpu_waiti_clk_gate = 1, \
-    .cali_ocode = 0\
+  .ck8m_wait = RTC_CNTL_CK8M_WAIT_DEFAULT, \
+  .xtal_wait = RTC_CNTL_XTL_BUF_WAIT_DEFAULT, \
+  .pll_wait  = RTC_CNTL_PLL_BUF_WAIT_DEFAULT, \
+  .clkctl_init = 1, \
+  .pwrctl_init = 1, \
+  .rtc_dboost_fpd = 1, \
+  .xtal_fpu = 0, \
+  .bbpll_fpu = 0, \
+  .cpu_waiti_clk_gate = 1, \
+  .cali_ocode = 0\
 }
 
+#ifdef CONFIG_RTC_DRIVER
+/* The magic data for the struct esp32c3_rtc_backup_s that is in RTC slow
+ * memory.
+ */
+
+#  define MAGIC_RTC_SAVE (0x11223344556677ull)
```

Comment:
```suggestion
#  define MAGIC_RTC_SAVE (UINT64_C(0x11223344556677))
```
nit: Use the definitions from `inttypes.h` for applying the suffix to integer constants

Reply:
Done